pages/a/zoomdata-xx.html

```diff
@@ -0,0 +1,64 @@
+title: Zoom Data XX
+link: <https://crate.io/a/zoomdata-xx>
+author: Chris Ward
+description: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+created: 2016/06/15
+post_name: zoomdata-xx
+status: publish
+post_type: post
+tags: xx
+category: news, developernews
+
+ZoomData helps you query and visualize your big data sets to gain valuable insights quickly and easily. The (beta) Crate connector supports all Crate data types, including objects.
+
+## Add Data Source
+
+Click the hamburger menu item on the left hand side of the Zoom Data dashboard, then click *Data Sources* and *Edit Data Sources* and select the Crate data source.
+
+![Add Data Source]({% media '/media/1606/zd-add-data-source.png' %})
+
+Give the connection a relevant name and click *Next*. Select the *Input New Credentials* radio button and add the JDBC url for nodes in your Crate instance, for example:
+
+```
+crate://NODE_ADDRESS1:4300,NODE_ADDRESS2:4300,etc
+```
+
+You can add as many nodes in the cluster as you want, a greater number helps with redundancy and fallbacks if one node becomes unavailable. Validate the connection, and click *Next*.
+
+In the next tab you can select the schema and individual fields for the data source, and ZoomData gives a preview of the data that will be loaded. On the next tab, you can override field names, types, and other details depending on the underlying schema.
+
+The next configuration tab let's you set when ZoomData should refresh your table data and schema, offering options to suit your needs and use case.
+
+In the final tab you can enable or disable the chart types you want available for this data source, and settings for each chart type.
+
+Click *Finish* and you can jump straight into the chart building screen.
```

Comment:
Not sure if these paragraphs could/should be reduced to something like "follow the guides and adjust the data sources as you go until you are at the chart building screen".  right now its kinda hard to follow?